Cargo.toml

```diff
@@ -26,6 +26,7 @@ strip = true
 [dependencies]
 anyhow = "1.0.68"
 api_client = { path = "api_client" }
+argh = "0.1.9"
```

Comment:
You forgot to remove the clap crate here.

Reply:
Ah yes of course. Both `ch-remote` and `cloud-hypervisor` binaries depend on this `Cargo.toml`.